cmd/fetch_repo/go_mod_download.go

```diff
@@ -0,0 +1,93 @@
+package main
+
+import (
+	"bytes"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"go/build"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"runtime"
+	"strings"
+)
+
+type GoModDownloadResult struct {
+	Dir   string
+	Sum   string
+	Error string
+}
+
+func isModeCacherwSupported() bool {
```

Comment:
Could you hardcode this to true and inline? Go 1.14 is super old, no need to keep supporting even older releases.

Reply:
Perfect, updated!